.github/workflows/release_apk.yml

```diff
@@ -0,0 +1,31 @@
+name: Test_and_build_artifacts_on_release
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build_apk_aab:
+    name: Build release artifacts
+    runs-on: ubuntu-latest
```

Comment:
Why ubuntu and not MacOS? I don't want to have MacOS. I simply want to understand pros and cons of using diffirent OS for testing and building?

Reply:
ubuntu is faster and it's crucial for using in actions because we have a limited amount of time

Reply:
Please add comment about it to code

Reply:
done